evm_arithmetization/src/poseidon/columns.rs

```diff
@@ -44,6 +71,17 @@ pub(crate) struct PoseidonColumnsView<T: Copy> {
     /// Holds the pseudo-inverse of (digest_high_limb_i - 2^32 + 1).
     pub pinv: [T; POSEIDON_DIGEST],
 
+    /// Holds the byte decomposition of the input, except for the less
+    /// significative byte.
+    pub input_bytes: [[T; FELT_MAX_BYTES - 1]; POSEIDON_SPONGE_RATE],
+
+    /// A flag indicating if this is a simple operation where inputs are
+    /// read from the top of the stack.
```

Comment:
for consistency with below:

```suggestion
    /// Indicates if this is a simple operation where inputs are
    /// read from the top of the stack.
```